tests/Game.class.test.js

```diff
@@ -0,0 +1,965 @@
+/* eslint-disable max-len */
+'use strict';
+
+const Game = require('../src/modules/Game.class');
+
+function sum(game) {
+  let result = 0;
+
+  for (const row of game.getState()) {
+    for (const n of row) {
+      result += n;
+    }
+  }
+
+  return result;
```

Comment:
The `sum` function could be simplified using `Array.prototype.reduce` for better readability and functional programming practices.